protos/bottle/core/v1/case.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package bottle.core.v1;
+
+import "bottle/fulfillment/v1/order.proto";
+
+message Case {
+  int64 timestamp = 1;
+  string requestId = 2;
+  string source = 3;
+
+  oneof bottle {
```

Comment:
Later we can use `Case` to wrap up other resources but for now the only "top level" one we have is an Order.

Reply:
Should we just call this something like `resource`?

Reply:
`resource` seems good and generic. Why is this named `Case`? Can we do something like `Message`?